website/src/css/custom.css

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+ :root {
+  --ifm-color-primary: #bf5934;
+  --ifm-color-primary-dark: rgb(172, 80, 47);
+  --ifm-color-primary-darker: rgb(162, 76, 44);
+  --ifm-color-primary-darkest: rgb(134, 62, 36);
+  --ifm-color-primary-light: rgb(201, 114, 82);
+  --ifm-color-primary-lighter: rgb(210, 139, 113);
+  --ifm-color-primary-lightest: rgb(223, 172, 154);
```

Comment:
Please revert these theme colors to the previous value `rgb(132, 49, 49)` and derived variants.